docs/features/software-catalog/external-integrations.md

```diff
@@ -688,3 +688,124 @@ export class SystemXReaderProcessor implements CatalogProcessor {
   }
 }
 ```
+
+### Supporting different metadata file formats
+
+Sometimes you might already have files in GitHub or some provider that Backstage already supports but the metadata format that you use is not the same as `catalog-info.yaml` files. In this case you can implement a custom parser that can read the files and convert them to the `Entity` format that Backstage expects, and it will integrate seamlessly into Catalog so that you can use things like the `GithubEntityProvider` to read these files.
```

Comment:
```suggestion
Sometimes you might already have files in GitHub or some provider that Backstage already supports but the metadata format that you use is not the same as `catalog-info.yaml` files. In this case you can implement a custom parser that can read the files and convert them on-the-fly to the `Entity` format that Backstage expects, and it will integrate seamlessly into Catalog so that you can use things like the `GithubEntityProvider` to read these files.
```

matter of taste but, thinking it may be worth to signal that it's not changing the original files